app/jobs/kithe/create_derivatives_job.rb

```diff
@@ -1,7 +1,12 @@
 module Kithe
   class CreateDerivativesJob < Job
     def perform(asset, mark_created: nil)
-      asset.create_derivatives(mark_created: mark_created)
+      begin
+        asset.create_derivatives(mark_created: mark_created)
+      rescue ActiveJob::DeserializationError
+        Rails.logger.error("Unable to derivatives for asset with ID #{asset.id}, as it was deleted first.")
```

Comment:
Maybe some weirder error happened than the asset being deleted, so message maybe should say "as the asset model was unavailable". 

I'm not totally sure that `asset.id` will be available in this condition, did you verify that?

There may be more info available in the `ActiveJob::DeserializationError` itself, if you change to `ActiveJob::DeserializationError => e`, you can get the exception in local variable `e`, and maybe actually print out it's representation in the error message too?

    Rails.logger.error("Unable to derivatives for asset with ID #{asset.id}, as it was not available: #{e}")

But make sure to actually repro at least in dev and see what the error looks like, and that it looks useful. To just record verification that we checked this, please put a copy paste of what final error looks like in this ticket -- or better yet, find a way to test it somehow! 

Perhaps make an asset, delete it, then manually trigger a CreateDerivativesJob for it. There is some hacky way to test Rails logger output in a test, but I'd have to google it.